### PR TITLE
bump JVM in Docker image

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Please don't depend on this, it breaks all the time.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ci/docker/daml-sdk/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk:8-jdk-hotspot
+FROM adoptopenjdk:11-jdk-hotspot
```
Comment (cocreature) [inline ci/docker/daml-sdk/Dockerfile:1]: This means we also use jdk8 in docker images for any pre 2.0 release we make now, e.g., SDK 1.18.0. Should we instead do a version check in the cron job and switch depending on that?

Comment (stefanobaghino-da) [reply ci/docker/daml-sdk/Dockerfile:1]: Should we have a 1.x set of images and a 2.x set of images for this purpose?

Comment (garyverhaegen-da) [reply ci/docker/daml-sdk/Dockerfile:1]: We run main branch YAML files but release commit source tree, so no, as I understand the script we'd still use Java 8 for the 1.18 branch.

Comment (cocreature) [reply ci/docker/daml-sdk/Dockerfile:1]: sounds plausible, thanks for checking

**Review by cocreature — APPROVED** [review]
Thank you!

File: ci/docker/daml-sdk/Dockerfile (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/docker/daml-sdk/Dockerfile:1]: Specifically [here](https://github.com/digital-asset/daml/blob/main/azure-cron.yml#L84-L86):
```
              git checkout "$version"
              cd ci/docker/daml-sdk
              docker build -t digitalasset/daml-sdk:${version#v} --build-arg VERSION=${version#v} .
```

Note that this means this PR will **not** fix the current image, only the next one.

Comment (SamirTalwar-DA) [reply ci/docker/daml-sdk/Dockerfile:1]: The `adoptopenjdk` images are now deprecated in favor of [`eclipse-temurin`](https://hub.docker.com/_/eclipse-temurin/). We might want to switch to `eclipse-temurin:11-jdk`.

Comment (garyverhaegen-da) [reply ci/docker/daml-sdk/Dockerfile:1]: Yes, thanks for raising.

**Pull request merged**